Makefile

```diff
@@ -367,6 +367,12 @@ fix-abci-app-specs:
 	echo "Successfully validated abcis!"
 
 
-AEA_AGENT:=valory/hello_world:latest:$(shell cat packages/hashes.csv | grep "agents/hello_world" | cut -d "," -f2 )
+AEA_AGENT_HELLO_WORLD:=valory/hello_world:latest:$(shell cat packages/hashes.csv | grep "agents/hello_world" | cut -d "," -f2 )
+AEA_AGENT_ORACLE:=valory/oracle:latest:$(shell cat packages/hashes.csv | grep "agents/oracle" | cut -d "," -f2 )
+AEA_AGENT_APY_ESTIMATION:=valory/oracle:latest:$(shell cat packages/hashes.csv | grep "agents/apy_estimation," | cut -d "," -f2 )
 release-images:
-	AEA_AGENT=${AEA_AGENT} skaffold build -p release && VERSION=latest AEA_AGENT=${AEA_AGENT} skaffold build -p release
+	export AEA_AGENT_ORACLE=${AEA_AGENT_ORACLE}
+	export AEA_AGENT_APY_ESTIMATION=${AEA_AGENT_APY_ESTIMATION}
+	export AEA_AGENT=${AEA_AGENT_HELLO_WORLD}
+	skaffold build -p release --cache-artifacts=false && VERSION=latest skaffold build -p release-latest
```

Comment:
The noCache flag gave me difficulties and did not reliably rebuild in a way i felt comfortable with.